src/runtime/google-analytics-event-listener.js

```diff
@@ -46,8 +46,8 @@ export class GoogleAnalyticsEventListener {
   handleClientEvent_(event, eventParams = undefined) {
     // Bail immediately if neither ga function (analytics.js) nor gtag function (gtag.js) exists in Window.
     if (
-      typeof this.win_.ga !== 'function' &&
-      typeof this.win_.gtag !== 'function'
+      !this.constructor.isGaEligible(this.deps_) &&
```

Comment:
I think `GoogleAnalyticsEventListener` would be more readable than `this.constructor`, thoughts?

Reply:
Agree, will change it to use the explicit class name.

Reply:
Thanks!